boardgame-ui/src/router.js

```diff
@@ -9,7 +9,7 @@ const guestOnly = async (to, from, next) => {
     next();
     return;
   }
-  next({name: 'home'});
+  next({name: 'login'});
```

Comment:
Why this change?

Reply:
see comment on currentUser

Reply:
OK, but better to remove it (it causes my browser to crash when I try to access the login page while authenticated, since it creates an infinite redirection loop)